### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Nowi5**
1. Fixing config/logger.php => config/logging.php
2. Explain difference between mysql and system connection
3. Example to get current tenant data - Get current Tenant #446

### CODE REVIEW

File: docs/hyn/5.2/logging.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Tenancy ships with a custom tenant aware logger which you can register in the lo
 - If no tenant is detected it will store the log file to:
 `storage/logs/`.
 
-Add a new `tenant` channel to the channels array in your `config/logger.php` like below:
+Add a new `tenant` channel to the channels array in your `config/logger.php` or rather `config/logging.php` for new Laravel version like below:
```
Comment (luceos) [inline docs/hyn/5.2/logging.md]: This seems to be a typo. It should have read `logging.php`. Please modify that and remove you addition.

Great find!

File: docs/hyn/5.2/installation.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ Under connections:
 ```
 
 Most often it makes sense to use either `system` or `tenant` as your default database connection setting; this is `DB_CONNECTION` in your `.env` file. This environment variable is assigned to the `default` key in the `config/database.php` configuration file.
+When setting the default database to system or tenant you do not need to use Laravel default `mysql` anymore. You can use for `mysql` and `system` the same settings, but it doesn't have to be.
```
Comment (luceos) [inline docs/hyn/5.2/installation.md]: Could you undo these changes, they have been discussed at length today here: https://github.com/tenancy/docs/pull/48

**Comment by Nowi5** [comment]
Thank you. Which other PRs do you mean?

[x]  move the "Get current environment" to the identification.md instead.
[x] reduce the length of that section title, eg "retrieve current tenant" nothing more?
[] The TenancyFacade can be more easily used from the root namespace with \Tenancy, showing both options makes sense. --> any open todo here, which?
[x] Getting the current tenant should be done using the new tenant() method on the Environment; I know it overlaps with website() but it makes talking about the tenant as being the website easier. --> added both as a lot still base on old way and should be easy to find via google
[x] The last example makes less sense, you can retrieve the current website of the currently identified hostname using the tenant or website method on the environment, there should be no need to complicate this further? --> Removed

File: docs/hyn/5.2/logging.md (hunk 1 above)
Comment (Nowi5) [reply docs/hyn/5.2/logging.md]: Thank you. Which other PRs do you mean?

[x] move the "Get current environment" to the identification.md instead.
[x] reduce the length of that section title, eg "retrieve current tenant" nothing more?
[] The TenancyFacade can be more easily used from the root namespace with \Tenancy, showing both options makes sense. --> any open todo here, which?
[x] Getting the current tenant should be done using the new tenant() method on the Environment; I know it overlaps with website() but it makes talking about the tenant as being the website easier. --> added both as a lot still base on old way and should be easy to find via google
[x] The last example makes less sense, you can retrieve the current website of the currently identified hostname using the tenant or website method on the environment, there should be no need to complicate this further? --> Removed

File: docs/hyn/5.2/installation.md (hunk 2 above)
Comment (Nowi5) [reply docs/hyn/5.2/installation.md]: For me this was the most challenging part to understand your concept. Is anything wrong on the statement? Otherwise I would prefer to keep it in as could be helpful for people like me which just get started.

File: docs/hyn/5.2/identification.md [hunk 3]
```diff
@@ -38,3 +38,29 @@ public function boot()
 
 > Be wary about switching tenants. I recommend switching only once during code
 execution. Use a background job to run mass changes on tenant databases.
+
+#### Retrieve current tenant
```
Comment (luceos) [inline docs/hyn/5.2/identification.md]: One last change 😊 

The 4 hashes is too much; it needs to be one 👍

File: docs/hyn/5.2/installation.md (hunk 2 above)
Comment (luceos) [reply docs/hyn/5.2/installation.md]: Your sentence still leaves too much open for interpretation:

 "You can use for `mysql` and `system` the same settings, but it doesn't have to be."

This could be interpreted in so many ways; stating "it doesn't have to be" only makes it harder to decide. I'd say stick to the previously improved sentence.

Thanks for jumping onto it, I understand the suggestion and there's much we can improve. If we get more of these suggestions I'll look into clarifying that full flow; I'm hoping the tutorial I'm writing will clarify some of the tenant/system choices as well. See: 
https://github.com/tenancy/docs/blob/dk/landing-tutorial/docs/hyn/5.2/tutorial-with-landing.md

Comment (Nowi5) [reply docs/hyn/5.2/installation.md]: However, done

File: docs/hyn/5.2/identification.md (hunk 3 above)
Comment (Nowi5) [reply docs/hyn/5.2/identification.md]: Done

**Comment by Nowi5** [comment]
Hi, I think all is done, please double check.
Thanks

**Comment by luceos** [comment]
Thank you for your patience!
